MAUI/Calendar/getting-started.md

```diff
@@ -1,20 +1,20 @@
 ---
 layout: post
 title: Getting started with MAUI Calendar control | Syncfusion
-description: Learn here all about getting started with Syncfusion .NET MAUI Calendar (SfCalendar) control and its basic features.
+description: Learn here all about getting started with Syncfusion .NET MAUI Calendar(SfCalendar) control and its basic features.
```

Comment:
Learn here all about getting started with Syncfusion .NET MAUI Calendar (SfCalendar) control and its basic features.

Reply:
Content changed in description